Public/javascripts/main.js

```diff
@@ -49,7 +49,7 @@ window.addEventListener('pageshow', function(event) {
   }
 })
 
-window.performSearch = _.debounce(function(searchQuery) {
```

Comment:
@finestructure Here was the original debounce call, but it was in slightly the wrong place..